src/ui/button/Button.scss

```diff
@@ -42,19 +54,17 @@
   }
 }
 
-.button {
-  padding: 9px 20px !important;
-
-  & > span {
-    @include label;
-
-    svg {
-      margin-right: 12px;
-    }
-  }
-}
-
 .iconButton {
   width: 50px;
   height: 50px;
+  display: inline-flex;
 }
+
+.textIcon{
+  display: flex;
+  align-items: center;
```

Comment:
instead lines 64 and 65 use one line `@include row`

Reply:
This style is temporary cause we need to merge all branches which use Button component for correct styling